js/platform-window.js

```diff
@@ -209,8 +217,10 @@ class TitleBar extends HTMLElement {
 
         });
 
-        fin.Platform.getCurrentSync().on('window-context-changed', evt => {
-            if (evt.context) {
+        fin.Platform.getCurrentSync().on('window-context-changed', async (evt) => {
+            console.log(evt);
```

Comment:
keep log?

Reply:
Remove imo. I had it in there mostly for debugging, but also to showcase the event payload shape as a context-specific demo. Makes sense to remove if we're merging into the general demo.